minecraft_servers/tubnet/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+  "server_name": "tubnet",
+  "nice_name": "TubNet",
+  "direct_ip": "tubnet.gg",
+  "server_wildcards": [
+    "%.tubnet.gg"
+  ],
+  "social": {
+    "web": "https://tubnet.gg",
+    "web_shop": "https://store.tubnet.gg",
+    "web_support": "https://forums.tubnet.gg/feeds",
+    "twitter": "TubNet",
+    "discord": "https://discord.gg/tubnet",
+    "youtube": "https://www.youtube.com/c/TubNet"
+  },
+  "gamemodes": {
+    "crystalrush": {
+      "name": "Crystal Rush", 
+      "url": "https://forums.tubnet.gg/forums/crystal-rush.7/"
+    },
+    "lightstrike": {
+      "name": "Light Strike", 
+      "url": "https://forums.tubnet.gg/forums/light-strike.8/"
+    },
+    "spacebound": {
+      "name": "Spacebound", 
+      "url": "https://forums.tubnet.gg/forums/spacebound.9/"
+    }
+  },
+  "discord": {
+    "server_id": 742980347540799508,
+    "rename_to_minecraft_name": false
+  },
+  "brand": {
+     "primary": "#ff9500",
+     "background": "#23b5f6",
+     "text": "#FFFFFF"
+  }, 
+  "location": {
+     "city": "Quebec",
+     "country": "Canada",
+     "country_code": "CA"
+  },
+  "yt_trailer": "5lKhhxS-KR0",
+  "user_stats": "https://forums.tubnet.gg/members/{userName}"

```

Comment:
<img width="471" alt="Bildschirm­foto 2022-12-19 um 13 52 05" src="https://user-images.githubusercontent.com/45363287/208430393-b77fd679-e62e-4d62-9f91-8ab8698034f3.png">

Reply:
I removed it!